docs/articles/posts/20241129_microsoft_hack.md

```diff
@@ -0,0 +1,52 @@
+---
+title: NHS x Microsoft Hack for Health
+authors: [AmaiaImazBlanco]
+date: 2024-12-04
+categories: 
+    - NHS Websites
+    - LLMs
+    - Python
+    - Events
+links:
+    - TBC
+slug: microsoft-hackathon
+description: >
+    The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended the Hack for Health at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. 
+---
+
+> The NHS England Data Science team, as well as a range of other analysts from across the organisation, attended an AI Hackathon at Microsoft, organised by the Data Science Team together with Microsoft and Kainos, with the key stakeholders being the NHS Websites Team. In this article, the author shares her experiences at the event. 
+
+<!-- more -->
+
+Last week, the long awaited Hack for Health hosted at the beautiful Story Club, in Paddington London, by Microsoft and NHSE finally happened! At the hackathon, every team had either a Kainos or Microsoft representative, as well as a range of participants from across NHS England, including a strong Data Science Team presence, with several of our team members in each group. And unlike a usual hackathon, this one had stakeholders and well defined use cases, we were all working to one common goal. 
+
+The use cases were outlined as: 
+
+1. **Search & Summarise**: using an extract of the NHS Corpus developed by one of our PhD interns, Sam Hollands, (we used two websites worth of data, out of 600 NHS websites, and it was already 25GB worth of data!), we had to develop an LLM that used RAG to sift through that data and retrieve answers to queries based on these NHS websites.
```

Comment:
Could link to RAG project for extra marketing points ;) https://nhsengland.github.io/datascience/our_work/ds251_RAG/